DigitalLearningSolutions.Data.Tests/DataServices/GroupsDataServiceTests.cs

```diff
@@ -528,6 +528,60 @@ public void GetGroupAtCentreById_returns_null_with_incorrect_centreId()
             result.Should().BeNull();
         }
 
+        [Test]
+        public void UpdateGroupName_updates_record()
+        {
+            using var transaction = new TransactionScope();
+            try
+            {
+                // Given
+                const int centerId = 101;
+                const int groupId = 5;
+                const string newGroupName = "Test group name";

```

Comment:
Can safely move this outside the transaction.

Reply:
I moved them out of try block